dmgbuild/core.py

```diff
@@ -391,14 +391,14 @@ def roundup(x, n):
                 for dirpath, dirnames, filenames in os.walk(path):
                     for f in filenames:
                         fp = os.path.join(dirpath, f)
-                        total_size += roundup(os.lstat(fp).st_size, 4096)
+                        total_size += roundup(os.lstat(fp).st_size, 4000)
```

Comment:
Changing this one to 4000 is definitely wrong. The block size on the disk is 4,096 bytes, so it really does need rounding up to a multiple of 4,096.